### PR TITLE
Save api keys

### PR DESCRIPTION
**Pull request by Feijk**
Add ability to save api-keys

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -23,29 +23,169 @@
...
+          <input class="form-control" id="twitter_consumer_key" name="twitter_consumer_key" type="text" placeholder="Twitter consumer key"></input>
+          <input class="form-control" id="twitter_consumer_secret" name="twitter_consumer_secret" type="text" placeholder="Twitter consumer secret"></input>
+          <input class="form-control" id="target_account" name="target_account" type="text" placeholder="Target account"></input>
+          <input class="form-control" id="my_account" name="my_account" type="text" placeholder="My account"></input>
+          <input class="form-control" id="follow_limit" name="follow_limit" type="text" placeholder="Follow Limit (default:100)"></input>
+        </div>
+        <div class="form-group">
+          <button id="start-following" class="btn btn-success">MARKETING, I SAID MARKETING MOETHERFUCKER!</button> <button class="btn btn-warning" id="clear-fields">Clear fields</button>
+          <div class="input-group">
+            <input class="form-control" id="setting-title" name="setting-title" type="text" placeholder="Title for saving (otpional)"></input>
```
Comment (nanuuki) [inline templates/index.html]: Small typo; `otpional` -> `optional`

File: templates/index.html [hunk 2]
```diff
@@ -23,29 +23,169 @@
...
+          </div><!-- /input-group -->
+          
+          
+        </div>
+        <div class="panel" id="response-field"></div>
+      </div>
+      <div class="col-sm-4">
+        <h3 class="settings-list-heading">Saved settings</h3>
+        <ul class="list-group" id="savedSettingsList">
+          <!-- <li class="list-group-item">
```
Comment (nanuuki) [inline templates/index.html]: Commented markup, oh noes! :boom:

**Comment by nanuuki** [comment]
+1 otherwise :+1:

**Comment by Feijk** [comment]
Fixed comments, merging now.

**Pull request merged**